README.md

```diff
@@ -84,11 +84,12 @@ specified, as in the above); the inner most level of indentation defines
 alterations to the general command or sub-tests. Each test command must
 define at least one sub-test:
 
-  * `status: <success|failure|<int>>`, where `success` and `failure` map to
-    platform specific notions of a command completing successfully or
-    unsuccessfully respectively and `<int>` is a signed integer checking for a
-    specific exit code on platforms that support it. If not specified,
-    defaults to `success`.
+  * `status: <success|failure|signal|<int>>`, where `success` and `failure` map
+    to platform specific notions of a command completing successfully or
+    unsuccessfully respectively. `signal` checks for termination due to a signal
+    on Unix platforms, on non-Unix platforms, the test will be ignored. `<int>`
```

Comment:
This looks good overall but, at the risk of being a grammar nit-picker, I think we need to change a comma to a semi-colon: "on Unix platforms; on non-Unix platforms".

Reply:
> at the risk of being a grammar nit-picker

I would never have pinned you as such! 

Fixed: 4038cbac4c04206c2a02fcb7c400a1b8d91f6876

Reply:
:)